app/views/people/_form.html.erb

```diff
@@ -14,9 +14,10 @@
     <%= f.input :end_date, as: :string,
                 input_html: { class: 'datepicker', size: 11,
                               value: format_date_value(f.object.end_date) } %>
-    <%= f.label :department_id %>
-    <br>
-    <%= f.collection_select :department_id, Department.managing_people.all, :id, :name, include_blank: true %>
+
+    <%= f.input :department_id, :collection => Department.managing_people.all,
+                :required => true, :selected => Department.managing_people.all.length == 1 ? Department.managing_people.all[0].id : nil %>
```

Comment:
This works, but would it be better to get some of this logic out of the view ? Also, this will fire even on edit. I don't think that's a good idea. I would rather only have the default value on new. We may enter people from a department, then turn it back to not managing_people. It's an edge case, but one I expect....

Reply:
Ah, yes, that's true. I'll work on it.